main.go

```diff
@@ -20,6 +22,31 @@ func main() {
 		return
 	}
 
+	latestVersion, err := utils.GetLatestVersion()
+	if err != nil {
+		fmt.Println(err)
+		return
```

Comment:
Bit late to the party, but wondering if we should run the commands assuming the version didn't change instead of returning here? If I'm not mistaken, this would make osdctl unusable if for whatever reason github isn't reachable.